### DISCUSSION TITLE
Failed to start check internet connection

### DISCUSSION THREAD

**Discussion by funkytraveler** [rshea0/enjinn#56] [Help] [answered]
After I click the 14 day trial button, I receive a 'Failed to start check internet connection' error. Internet is working correctly. Running on M1 Mac mini running Bg Sur (v11.4).

**Comment by shayded-exe** [comment]
So sorry for the issue. Please email me your license key at shayded@shayded.com and I'll send you an activated license file.

**Comment by shayded-exe** [comment, marked as answer]
This has been fixed in [v2.2.0](https://github.com/rshea0/tuneup-prime/releases/tag/v2.2.0)
